apollo-idling-resource/src/test/java/com/apollographql/apollo/test/espresso/ApolloIdlingResourceTest.kt

```diff
@@ -0,0 +1,210 @@
+package com.apollographql.apollo.test.espresso
```

Comment:
Converted a bunch of files to Kotlin as I'd like to get rid of `@JvmField` annotation for `3.0.0-alpha01`